source/MaterialXTest/MaterialXRender/RenderUtil.cpp

```diff
@@ -134,24 +134,11 @@ bool ShaderRenderTester::validate(const mx::FilePathVec& testRootPaths, const mx
 
     mx::ScopedTimer ioTimer(&profileTimes.ioTime);
     mx::FilePathVec dirs;
-    if (options.externalTestPaths.size() == 0)
+    for (const auto& root : options.renderTestPaths)
```

Comment:
Never mind about my previous comment about what get's run.
It is as assumed that only these paths are considered for testing now.
Much more flexible and does not force running the testsuite files.